lib/tasks/migrate_data.rake

```diff
@@ -99,4 +99,36 @@ namespace :tww do
       end
     end
   end
+
+  desc "migrate some test data from quickbooks"
+  task migrate_organizations_to_qbo: :environment do
+    qb_connection = Division.root.qb_connection
+
+    customers = Quickbooks::Service::Customer.new(qb_connection.auth_details).all
+
+    customer_hash = {}
+    customers.each do |customer|
+      # Customer does not have a name property so we use display name since that is
+      # what is usually show in the QBO UI.
+      customer_hash[customer.display_name] = customer
+    end
```

Comment:
Note that the above could be written simply as `customers_hash = customers.index_by(&:display_name)`.